swagger_to_sdk_config.json

```diff
@@ -5,494 +5,1501 @@
+    },
+    "azure_mgmt_recovery_services_backup_2017_07_01": {
+      "autorest_options": {
+        "namespace": "Azure::ARM::RecoveryServicesBackup::Api_2017_07_01",
+        "package-version": "0.11.0",
+        "package-name": "azure_mgmt_recovery_services_backup",
+        "input-file": [
+          "specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/2017-07-01/jobs.json"
+        ]
       },
-      "output_dir": "management/azure_mgmt_recovery_services/lib",
-      "wrapper_filesOrDirs": [
-        "azure_mgmt_recovery_services.rb"
-      ]
+      "output_dir": "management/azure_mgmt_recovery_services_backup/lib/2017-07-01",
+      "generated_relative_base_directory": "2017-07-01",
+      "build_dir": "management/azure_mgmt_recovery_services_backup/lib/2017-07-01"
     },
-    "azure_mgmt_recovery_services_backup": {
+    "azure_mgmt_recovery_services_backup_2016_12_01": {
+      "autorest_options": {
+        "namespace": "Azure::ARM::RecoveryServicesBackup::Api_2016_12_01",
+        "package-version": "0.11.0",
+        "package-name": "azure_mgmt_recovery_services_backup",
+        "input-file": [
+          "specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/2016-12-01/backupManagement.json"
+        ]
+      },
+      "output_dir": "management/azure_mgmt_recovery_services_backup/lib/2016-12-01",
+      "generated_relative_base_directory": "2016-12-01",
+      "build_dir": "management/azure_mgmt_recovery_services_backup/lib/2016-12-01"
+    },
+    "azure_mgmt_recovery_services_backup_2016_08_10": {
+      "autorest_options": {
+        "namespace": "Azure::ARM::RecoveryServicesBackup::Api_2016_08_10",
+        "package-version": "0.11.0",
+        "package-name": "azure_mgmt_recovery_services_backup",
+        "input-file": [
+          "specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/2016-08-10/operations.json"
+        ]
+      },
+      "output_dir": "management/azure_mgmt_recovery_services_backup/lib/2016-08-10",
+      "generated_relative_base_directory": "2016-08-10",
+      "build_dir": "management/azure_mgmt_recovery_services_backup/lib/2016-08-10"
+    },
+    "azure_mgmt_recovery_services_backup_2016_06_01": {
       "markdown": "specification/recoveryservicesbackup/resource-manager/readme.md",
       "autorest_options": {
-        "namespace": "Azure::ARM::RecoveryServicesBackup",
+        "namespace": "Azure::ARM::RecoveryServicesBackup::Api_2016_06_01",
+        "package-version": "0.11.0",
         "package-name": "azure_mgmt_recovery_services_backup",
-        "package-version": "0.11.0"
+        "tag": "package-2016-06"
+      },
+      "output_dir": "management/azure_mgmt_recovery_services_backup/lib/2016-06-01",
+      "generated_relative_base_directory": "2016-06-01",
+      "build_dir": "management/azure_mgmt_recovery_services_backup/lib/2016-06-01"
+    },
+    "azure_mgmt_recovery_services_2016_08_10": {
```

Comment:
should we call this recovery_services_site_recovery? I think we should update it in the config.json, thoughts?

Reply:
good idea